### PR TITLE
Global executable

### PR DESCRIPTION
**Pull request by Mr-Sunglasses**
Now you can execute the Autotype CLI from anywhere in the Commandline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,32 @@ Provide the path of the file to be autotyped and the delay time through teminal/
 
 `python3 command_line_script --path filePath --delay delay_before_typing`
 
+### Run it Global Commandline Executable
+
+If you are a Linux or a Macos user just all AutoType to Your .zshrc or .bashrc file
```
Comment (tushar5526) [inline README.md:31]: Typo ? all -> add

File: Autotype [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python3
+import os
+import typer
+from typing import Optional
+from Simulator import Type
+import time
+
+def autotype_cli(
```
Comment (tushar5526) [inline Autotype:8]: Does this seem to be redundant cant we import `autotype_cli` from `command_line_script.py`?

Comment (Mr-Sunglasses) [reply Autotype:8]: So Basically this file is a Unix-executable-file. So, the thing is, it should be standalone from the `command_line_script.py`. Technically the Right way to structure these executable files was to put them all in a `bin` folder but, Autotype is having some dependency problems.

Comment (tushar5526) [reply Autotype:8]: I think I am a bit confused.

I tried making the `command_line_script.py` and `GUI_script.py` executable by adding the shebang `#! /usr/bin/env python3` to both these files and make them executable `chmod +x` 

After exporting the paths in the current shell, I was able to call `command_line_script.py` and `GUI_script.py` directly (from anywhere in my filesystem in that shell) without appending them with `python` (shebang does the job)

So either `Autotype` could take args like `cli and gui` and call the respected functions accordingly.

If its still confusing we can get on a meet and clear this out. 
Let me know your thoughts

**Pull request closed without merging**